cmd/wallet-web/src/components/SkeletonLoader/CredentialPreviewSkeletonComponent.vue

```diff
@@ -0,0 +1,40 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div class="flex flex-col justify-start items-start px-0 w-full">
+    <div
+      class="flex z-10 flex-row justify-between items-center lg:py-6 pr-8 pl-5 w-full lg:h-24 bg-neutrals-whiteLilac rounded-xl"
+      :class="isTall ? 'h-24 py-6' : 'h-20 py-5'"
```

Comment:
where is this isTall defined ?

Reply:
I defined it as a prop, set to `false` by default and passed in `true` for the Credential Details Page. On the Credential Details Page, `isTall` is `true` for any screen size. On the Credentials Page, `isTall` is only `true` for large screens (this is based on the designs)